homeassistant/components/http/ban.py

```diff
@@ -241,8 +275,29 @@ def _add_ban(self, ip_ban: IpBan) -> None:
             # Write in a single write call to avoid interleaved writes
             out.write("\n" + yaml.dump(ip_))
 
+    def _write_bans(self) -> None:
+        """Write IP bans to config file."""
+        ip_bans = {
+            str(ip_ban.ip_address): {ATTR_BANNED_AT: ip_ban.banned_at.isoformat()}
+            for ip_ban in self.ip_bans_lookup.values()
+        }
+
+        with open(self.path, "r+", encoding="utf8") as out:
+            out.seek(0)
+            out.truncate()
+            out.write("\n" + yaml.dump(ip_bans))
```

Comment:
Might be better to use `save_yaml` here?

https://github.com/home-assistant/core/blob/f0077ac27e4bb7909b8f39698d5c012218d5aeeb/homeassistant/util/yaml/dumper.py#L31-L33

Reply:
Hm could we use yes, but I try to have the same out as before (`\n` between the ban blocks)